tests/MongoDB.Driver.Examples/DocumentationExamples.cs

```diff
@@ -1568,6 +1568,46 @@ public void Index_Example_2()
             result.Should().Be("cuisine_1_name_1");
         }
 
+        [Fact]
+        public void Causal_Consistency_Example_2()
+        {
+            RequireServer.Check().SupportsCausalConsistency();
+
+            this.client.GetDatabase("myDatabase").DropCollection("myCollection");
```

Comment:
I don't like that we have two variables with the same name, "this.client" and "client". Maybe we have no choice...

Reply:
I've placed this test to the new CausalConsistencyExamples.cs file. So this is no longer a problem